src/common/lib/keymaps.c

```
@@ -23,95 +23,190 @@
- *
- * If the key code is KC_NFLP, it searches the keymap again to get the flip key code and converts it
- * to the corresponding numpad flip code. Finally, it returns the key code.
- *
- * @param pos  The position of the key in the keymap.
- * @param make A boolean indicating whether the key is being pressed (true) or released (false).
- *
- * @return The key code at the specified position in the keymap.
+ * @return The HID keycode to send, or KC_NO if consumed by layer operation.
  */
-uint8_t keymap_get_key_val(uint8_t pos, bool make) {
+uint8_t keymap_get_key_val(uint8_t pos, bool make, bool* suppress_shift) {
```

Cppcheck (2.19.0)

[style] The function 'keymap_get_key_val' is never used.

(unusedFunction)